packages/flutter/lib/painting.dart

```diff
@@ -35,7 +35,7 @@ export 'src/painting/colors.dart';
 export 'src/painting/continuous_rectangle_border.dart';
 export 'src/painting/debug.dart';
 export 'src/painting/decoration.dart';
-export 'src/painting/decoration_image.dart';
+export 'src/painting/decoration_image.dart' hide ImageTilingInfo, createTilingInfo;
```

Comment:
Our general policy is to just export these all. They are reachable, so there isn't really a point in hiding them.

Reply:
Done